Remove scripts because these contain additional, undocumented dependencies and
are not required as the package is used only as a library.

--- setup.cfg.orig	2020-12-02 20:52:12 UTC
+++ setup.cfg
@@ -25,10 +25,10 @@ classifier = 
 [files]
 packages = 
 	openstackdocstheme
-scripts = 
-	bin/docstheme-build-translated.sh
-	bin/docstheme-lang-display-name.py
-	bin/docstheme-build-pdf
+#scripts = 
+#	bin/docstheme-build-translated.sh
+#	bin/docstheme-lang-display-name.py
+#	bin/docstheme-build-pdf
 
 [egg_info]
 tag_build = 
